floodviz/views.py

```diff
@@ -7,26 +7,37 @@
 from . import map_utils
 from . import peak_flow_utils
 from . import REFERENCE_DATA as ref
+from .linked_data_utils import LinkedData
 
 url_nwis_prefix = app.config['NWIS_SITE_SERVICE_ENDPOINT']
+linked_data = LinkedData()
```

Comment:
Setters vs passing things like bbox and dates to the actual object. I'm not sure which is preferred here, but just noting the other option.